docs/articles/documentation/using-testcafe/common-concepts/using-browsers.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: docs
+title: Using Browsers
+permalink: /documentation/using-testcafe/common-concepts/using-browsers.html
+---
+# Using Browsers
+
+TestCafe allows you to easily set up testing in different web browsers: a local, remote, or custom browser supplied by a [browser provider plugin](#browser-provider-plugins).
+You just need to specify them in one of two ways:
+
+* through the command line (see [Browser List](../command-line-interface.md#browser-list))
+* programmatically (by using the [runner.browsers](../programming-interface/runner.md#browsers) method)
+
+## Local Browser Names
+
+To identify local browsers, browser short names are used. They allow you to avoid dealing with paths to browser executables or other awkward ways to refer to a browser.
+
+The following table lists all available browser names.
+
+Browser           | Alias
+----------------- | ----------
+Chromium          | `chromium`
+Google Chrome     | `chrome`
+Internet Explorer | `ie`
+Microsoft Edge    | `edge`
+Mozilla Firefox   | `firefox`
+Opera             | `opera`
+Safari            | `safari`
+
+To obtain the list of browsers installed on the current machine, run TestCafe
+from the command line with the [-b option](../command-line-interface.md#-b---list-browsers).
+
+## Browser Provider Plugins
+
+You can create a [custom browser provider plugin](../../extending-testcafe/custom-browser-provider-plugin/) to use a specific browser for testing.
```

Comment:
Well, you can use a `specific` browser without a browser provider. Please be more *specific* on when we need to use the browser provider.